deployment/lib/lambdas/cf-url-rewriter.ts

```diff
@@ -0,0 +1,70 @@
+import { CloudFrontRequestEvent, CloudFrontRequest } from 'aws-lambda';
+
+const https = require('https');
+
+export async function handler(event: CloudFrontRequestEvent, context, callback): Promise<CloudFrontRequest> {
+  const request = event.Records[0].cf.request;
+
+  console.log('request', JSON.stringify(request));
+
+  if (request.uri.includes("latest")) {
+
+    const indexUri = request.uri.replace(/latest.*/, 'dist/index.json');
+
+    const data = await httpGet('https://' + request.headers.host[0].value + indexUri);
```

Comment:
1. What happens if the file doesn't exist? The promise is rejected and you get an error?
2. I wouldn't `await` here and use `.then` so that the call is not blocking.

Reply:
1. Added a try/catch for error handling.
2. Since the logic depends on the content of the index file, I think it makes sense to be a blocking call. Let me know if I misunderstand your comment.